generators/client/templates/angular/src/main/webapp/app/admin/user-management/_user-management.component.ts

```diff
@@ -132,7 +144,20 @@ export class UserMgmtComponent implements OnInit {
         this.users = data;
     }
 
-    private onError (error) {
+    private onError(error) {
         this.alertService.error(error.error, error.message, null);
     }
+
+    createUser() {
+        this.modalRef = this.userModalService.open(UserMgmtDialogComponent);
```

Comment:
That was exactly what I had in mind. (tried too)
This in turn makes us to create that thin component for every logical entity / modal we need to replicate this. This needs some exploration too.